pipelinewise/cli/multiprocess.py

```diff
@@ -0,0 +1,30 @@
+import multiprocessing
+import traceback
+
+
+class MultiProcess(multiprocessing.Process):
```

Comment:
why is this named `MultiProcess`? what's multi about it?

Reply:
- [ ] rename it to Process